le_utils/resources/presetlookup.json

```diff
@@ -234,11 +234,23 @@
 		"thumbnail": false,
 		"subtitle": false,
 		"display": true,
-		"order": 2,
-		"kind": "html5",
+		"order": 1,
+		"kind": "h5p",
 		"allowed_formats": ["h5p"],
 		"convertible_formats": []
 	},
+	"h5p_thumbnail": {
```

Comment:
Could you please set the h5p format preset `display` field to false? This will hide it from Studio's uploading option